src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -795,6 +772,88 @@ void testErrorReportedForMissingUpdateEntityNameField() {
         assertError(qualifiedFieldName, validationMessage, errors);
     }
 
+    private void testFullUpdateRemoveValidationWithoutTrusts() {
```

Comment:
These methods have simply moved and been made private non-test methods so that they can be called from both the Update and Remove unit tests.